Adding a paragraph to clarify environment variable setting (#2357)
saucelabs/sauce-docs: docs/orchestrate/quickstart-playwright.md

1. Enter your Sauce Labs Username and Access Key at the prompts.

:::note 

> The `saucectl configure` command will not actually set system environment variables, nor will it alter your .bashrc or other command-line profiles to set them by default. To do this, you will need to run `export` commands as specified in our [environment variables](/basics/environment-variables/#setting-up-environment-variables) page.

Reviewer: ```suggestion
The `saucectl configure` command will not set system environment variables or alter your .bashrc or other command-line profiles to set them by default. To do this, you must run `export` commands as specified in our [environment variables](/basics/environment-variables/#setting-up-environment-variables) page.
```
